pkgs/development/libraries/science/math/sympow/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, fetchurl
+, fetchpatch
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sympow";
+  version = "1.018.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    # Original website no longer reachable
+    url = "http://mirrors.mit.edu/sage/spkg/upstream/sympow/sympow-${version}.tar.bz2";
+    sha256 = "0hphs7ia1wr5mydf288zvwj4svrymfpadcg3pi6w80km2yg5bm3c";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  configurePhase = ''
+    ./Configure # doesn't take any options
+  '';
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    mkdir -p "$out/share/sympow"
+    cp -r *.gp datafiles "$out/share/sympow"
+    install -Dm755 sympow "$out/share/sympow/sympow"
+    cp new_data "$out/bin/new_data"
+
+    makeWrapper "$out/share/sympow/sympow" "$out/bin/sympow" \
+      --run 'export SYMPOW_LOCAL="$HOME/.local/share/sympow"' \
+      --run 'if [ ! -d "$SYMPOW_LOCAL" ]; then
+        mkdir -p "$SYMPOW_LOCAL" 
+        cp -r ${placeholder "out"}/share/sympow/* "$SYMPOW_LOCAL"
+        chmod -R +xw "$SYMPOW_LOCAL"
+    fi' \
+      --run 'cd "$SYMPOW_LOCAL"'
+  '';
+
+  patches = [
+    # don't hardcode paths
```

Comment:
Instead of a comment, you can pass ```name = "do_not_hardcode_paths.patch``` to ```fetchpatch``` which also makes it easier to work with.

Reply:
I added names (but also kept the comments, as they offer more freedom for explanation than a filename).

Reply:
Documentation is always good!